src/containers/BrowseMatch/index.js

```diff
@@ -170,72 +179,72 @@ class BrowseMatch extends React.Component
                 </Link>
             </Typography>
         </ListItem>)
-    }
+    }*/
 
-    getMatchesList(item, classes)
-    {
+    getMatchesList(item, classes) {
         const languageTooltip = 'Matches are sorted by compatibility relative to your language preferences. ' +
                                 'Matches on the left are rated higher with a descending compatibility going right.';
 
         return (
-                <div key={item.languageName}>
-                    <ExpansionPanel defaultExpanded={this.state.isDefaultExpand}>
-                        <ExpansionPanelSummary
-                            expandIcon={<ExpandMoreIcon />}
-                            aria-controls="panel1a-content"
-                            id="panel1a-header"
-                        >
-                        <Typography className={classes.heading}>
-                            Possible matches who can teach you {item.languageName} - <strong>{item.matches.length} match(es) &nbsp;&nbsp;&nbsp;&nbsp;</strong>
+            <div key={item.languageName}>
+                <ExpansionPanel className={classes.expansionPan} defaultExpanded={this.state.isDefaultExpand}>
+                    <ExpansionPanelSummary
+                        expandIcon={<ExpandMoreIcon />}
+                        aria-controls="panel1a-content"
+                        id="panel1a-header"
+                    >
+                        <Typography variant="h6" className={classes.headingxxxxx}>
```

Comment:
this class name? headingxxxxx

Reply:
The heading class does no visual changes. The code is full of unnecessary style declarations. I will go through them and delete these classes